runtime/compiler/runtime/IProfiler.cpp

```diff
@@ -4087,7 +4087,8 @@ TR_IProfiler::processWorkingQueue()
          {
 #if defined(J9VM_OPT_CRIU_SUPPORT)
          // Check if the IProfiler Thread should be suspended for checkpoint
-         if (_compInfo->getJITConfig()->javaVM->internalVMFunctions->isCheckpointAllowed(_iprofilerThread))
+         J9JavaVM *vm = _compInfo->getJITConfig()->javaVM;
+         if (vm->internalVMFunctions->isCheckpointAllowed(vm))
```

Comment:
Suggest the name of the new local should be `javaVM` like was done in `VMJ9.cpp` and for consistency with line 4191.

Reply:
Renamed